stm/stm32l4xx/stm32l4xx_hal_init.c

```diff
@@ -4,20 +4,17 @@
  * @brief This function is executed in case of error occurrence.
  * 
  */
-static void Error_Handler(void)
-{
+__attribute__((weak)) void Error_Handler(void) {
```

Comment:
A bit pedantic, but why not do __weak like you did in the f4 file?

Reply:
oh shit i forgor